pkg/scheduler/factory/factory.go

```diff
@@ -213,25 +210,17 @@ type ConfigFactoryArgs struct {
 	PercentageOfNodesToScore       int32
 	BindTimeoutSeconds             int64
 	StopCh                         <-chan struct{}
-	Registry                       framework.Registry
-	Plugins                        *config.Plugins
-	PluginConfig                   []config.PluginConfig
```

Comment:
So the only reason for these items in the config was to later create the framework, right?

Reply:
yes, couldn't find them referenced anywhere else